en/docs/reference/connectors/file-connector/file-connector-config.md

```diff
@@ -3479,6 +3749,36 @@ The following operations allow you to work with the File Connector version 4. Cl
         </tr>
     </table>
 
+    **Sample configuration**
+    ```xml
+    <file.write configKey="CONNECTION_NAME">
+        <filePath>{$ctx:filePath}</filePath>
+        <contentOrExpression>{$ctx:contentOrExpression}</contentOrExpression>
+        <mimeType>{$ctx:mimeType}</mimeType>
+        <writeMode>{$ctx:writeMode}</writeMode>
+        <appendNewLine>{$ctx:appendNewLine}</appendNewLine>
+        <encoding>{$ctx:encoding}</encoding>
+        <compress>{$ctx:compress}</compress>
+        <enableStreaming>{$ctx:enableStreaming}</enableStreaming>
+        <enableLock>{$ctx:enableLock}</enableLock>
+        <includeResultTo>{$ctx:includeResultTo}</includeResultTo>
+        <resultPropertyName>{$ctx:resultPropertyName}</resultPropertyName>
+        <updateLastModified>{$ctx:updateLastModified}</updateLastModified>
+        <maxRetries>{$ctx:maxRetries}</maxRetries>
+        <retryDelay>{$ctx:retryDelay}</retryDelay>
+    </file.write>
+    ```
+    
+    **Sample request**
+
+    Following is a sample REST/JSON request that can be handled by the append operation.
+    ```json
+        {
+            "filePath":"/home/vive/Desktop/file.test.txt",
+            "contentOrExpression":"Message Body"
```

Comment:
This can mislead. Basically this has to be configured from where the content should be fetched. So basically can't think of a usecase where we pass this in the request

Reply:
Removed this.